apps/api/src/domain/models/audio-visual/audio-item/repositories/arango-audio-item-query-repository.spec.ts

```diff
@@ -412,4 +414,71 @@ describe(`ArangoAudioItemQueryRepository`, () => {
             expect(missingLineItems).toEqual([]);
         });
     });
+
+    describe(`translate line item`, () => {
+        const originalLanguageCode = LanguageCode.Chilcotin;
+
+        const translationLanguageCode = LanguageCode.English;
+
+        const translationText = 'this is how to translate what was said to English';
+
+        const inPointMilliseconds = 100;
+
+        const outPointMilliseconds = 200;
+
+        const participant = buildTestInstance(TranscriptParticipant);
+
+        const targetLineItem = buildTestInstance(TranscriptItem, {
+            inPointMilliseconds,
+            outPointMilliseconds,
+            speakerInitials: participant.initials,
+            text: buildMultilingualTextWithSingleItem('original text', originalLanguageCode),
+        });
+
+        const existingTranscript = buildTestInstance(Transcript, {
+            participants: [participant],
+            items: [targetLineItem],
+        });
+
+        const targetAudioItem = buildTestInstance(EventSourcedAudioItemViewModel, {
+            transcript: existingTranscript,
```

Comment:
Should we add lengthMilliseconds and make it greater than the outpoint? I also wonder if we should ignore out of range writes, although there's no way this would happen given our command validation before this event was emitted.

Reply:
done